Connection.php

```diff
@@ -555,3 +555,26 @@ public function close($code = self::CLOSE_NORMAL, $reason = null)
         return $connection->disconnect();
     }
 }
+
+/**
+ * Register socket wrappers
```

Comment:
What do you think about moving this code as a static method on the `Hoa\Websocket\Connection` class? Would it make sense?